src/Expander.hs

```diff
@@ -984,7 +986,7 @@ initializeKernel = do
       bind b val
       addToKernel name runtime b
 
-    addDeclPrimitive :: Text -> (Scope -> DeclPtr -> DeclValidityPtr -> Syntax -> Expand ()) -> Expand ()
+    addDeclPrimitive :: Text -> (Scope -> ModBodyPtr -> DeclValidityPtr -> Syntax -> Expand ()) -> Expand ()
```

Comment:
Currently all the primitives bind a single `Decl`, so this `DeclPtr` doesn't really need to change to a `ModBodyPtr` yet. But soon we'll have `no-decls` and `decls` primitives which do require a `ModBodyPtr`.

Reply:
Yep :-) This is exactly how I expected the code to turn out.